app/src/main/java/com/github/sdp/mediato/data/Database.java

```diff
@@ -214,4 +221,45 @@ public static CompletableFuture<String> addCollection(String username, Collectio
         return future;
     }
 
+    /**
+     * Method to follow a user
+     *
+     * @param myUsername       the current user's username
+     * @param usernameToFollow the username of the user to follow
+     */
+    public static void followUser(String myUsername, String usernameToFollow) {
+        CompletableFuture<String> future = new CompletableFuture<>();

```

Comment:
Future is never used, consider returning it and adding exception handling with it (for example when one of the user is never found).

Reply:
Thanks for pointing that out. I forgot to remove it (adressed in a commit).